web/src/main/webapp/new-error.jsp

```diff
@@ -68,6 +68,13 @@
       <wc-footer/>
     </c:when>
 
+    <c:when test="${param.reason eq 'another-session-active'}">
```

Comment:
See comment in the common PR if we should change the reason to 'invalid-sessionid' or something else.

Reply:
"invalid-launchid"